### PR TITLE
Homepage UI Improvements

### PR DESCRIPTION
**Pull request by faithcorinne**
We will need to revisit the actual desired UI (working without mocks atm), but for now I have implemented:
- Full width lead with search bar (non-functional at the moment)
- Jumbotron "How It Works" section
- Temporary footer (will require refactor)

### CODE REVIEW

File: src/components/HomePage.js [hunk 1]
```diff
@@ -5,64 +5,79 @@ const HomePage = () => {
...
-                  <input id="search-bar" type="search" />
+            {/*Lead Section w/ Search*/}
+              <div className="container-fluid">
+                <div className="row">
+                  <div className="col-lg-12 align-center" id="lead">
+                    <h1>Hoppi Space</h1>
+                    <h2>Connecting the artist within to the space you are without</h2>
+                  {/*TO-DO: Add Anchor linked buttons to content below*/}
+                  <form>
+                      <div class="col-lg-12">
```
Comment (pearlsortman) [inline src/components/HomePage.js:16]: Everywhere you have class needs to be className

File: src/components/HomePage.js [hunk 2]
```diff
@@ -5,64 +5,79 @@ const HomePage = () => {
...
-                  <li>Support: support@hoppispace.com</li>
-                  <li>{/*TO DO: Add phone icon*/}Phone: 000.000.0000</li>
-                </ul>
+            {/*How it Works Section*/}
+              <div className="container">
+                <div className="jumbotron" id="how-it-works">
+                {/*TO DO: Ad back countraction to "you've"*/}
+                  <h2 className="display-3">How It Works</h2>
+                  <h3>Browse, Book, Pay</h3>
+                  <hr/>
```
Comment (pearlsortman) [inline src/components/HomePage.js:44]: Meant to be <br />?

Comment (pearlsortman) [reply src/components/HomePage.js:44]: Oops it did an actual break. Should this be a br tag?

File: src/css/styles.css [hunk 3]
```diff
@@ -1,48 +1,59 @@
...
 #footer-items {
-  background-color: #35a7ff;
-  color: #fff; }
+  background-color: $secondary-color;
+  color: #fff;
+}
+
+#lead {
+  background-color: #777;
+}*/
```
Comment (pearlsortman) [inline src/css/styles.css:57]: Why is all this commented out?

File: src/styles/styles.scss [hunk 4]
```diff
@@ -7,8 +7,8 @@ body {
   background-color: $body-background-color;
   line-height: 1.4em;
   color: #4d4d4d;
-  min-width: 230px;
-  max-width: 85%;
+  //min-width: 85%;
+  //max-width: 85%;
```
Comment (pearlsortman) [inline src/styles/styles.scss:11]: Can delete altogether

**Review by pearlsortman — CHANGES_REQUESTED** [review]
See in-line comments.

File: src/components/HomePage.js (hunk 2 above)
Comment (faithcorinne) [reply src/components/HomePage.js:44]: This is a horizontal rule. That's expected 👍

File: src/css/styles.css (hunk 3 above)
Comment (faithcorinne) [reply src/css/styles.css:57]: Compiling issue between my SCSS and CSS; does not effect the work but it'sa build thing I need to figure out for the future.

**Review by faithcorinne — COMMENTED** [review]
Requested changes made, please review.

**Review by pearlsortman — APPROVED** [review]

**Pull request merged**